Resources/Prototypes/_Impstation/Entities/Mobs/Customization/Markings/misc.yml

```
@@ -0,0 +1,903 @@
+##i did markings for a lot of different species so for now theyre are all here##
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)